lib/log_stuff.rb

```diff
@@ -65,7 +65,7 @@ def self.compile_fields_and_tags(args)
     return local_fields, local_tags
   end
 
-  %w( fatal error warn info debug ).each do |severity|
+  ["fatal", "error", "warn", "info", "debug"].each do |severity|
```

Comment:
how about to also use single quotes like the other changes?

Reply:
well spotted and done.